Restructuring the report & HCI Theory (#29)
chhoumann/p8: report/src/problem_analysis/hci/designing_products.tex

\section{Designing a product} % generel hci
When establishing a design for a product, one often begins with exploring initial thoughts and ideas for a design, only to then present the design to the user, make adjustments, and repeat the process. 
This can be described in a process called the design funnel.\cite{BUXTON2007135_skething}

> Buxton\cite{BUXTON2007135_skething,BUXTON2007139_prototyping} describes how to move from an exploratory analysis of different designs through sketching to interactive, product refinement using prototypes. 

Reviewer: Use \citeauthor instead?

Reviewer 2: ```suggestion
Buxton\cite{BUXTON2007135_skething,BUXTON2007139_prototyping} describes how to move from an exploratory analysis of different designs through sketching to interactive product refinement using prototypes. 
```

Sketching is the act of ideation and capturing key concepts of the design in a cheap and easy-to-make artifact. 

> It is important to notice that the artifact itself is but a byproduct of the process, while the goal of the activity lies within the process itself.\cite{BUXTON2007135_skething}

Reviewer: ```suggestion
It is important to note that the artifact itself is a byproduct of the process, while the goal of the activity lies within the process itself.\cite{BUXTON2007135_skething}
```

> The true goal is to understand user needs and develop a design concept that both the designer and user find agreeable in terms of requirements and user experience. 

Reviewer: ```suggestion
The goal is to understand user needs and develop a design concept that both the designer and user find agreeable in terms of requirements and user experience. 
```

> When sketching, one must put aside assumptions and preconceptions about the product one is designing.

Reviewer: ```suggestion
When sketching, one must put aside assumptions and preconceptions about the product being designed.
```

> This helps the designer discover new ideas for the product due to not being constrained by an existing understanding of requirements, user needs, and expert knowledge. %kilde? 

Reviewer 2: Either you cite or you don't comment that you need a cite. No todos!

```suggestion
This helps the designer discover new ideas for the product due to not being constrained by an existing understanding of requirements, user needs, and expert knowledge.
```

Reviewer: ```suggestion
This helps discovering new ideas for the product from not being constrained by an existing understanding of requirements, user needs, and expert knowledge.
```

Sketching techniques often include the participation of users of the developing product.

> This helps the designer focus efforts on designing a system that suits the user and fulfills their needs.

Reviewer: ```suggestion
This helps the designer focus efforts on designing a system that fulfills user needs.
```

This is known as user-centered design (UCD). \cite{user-centred-design}
Techniques within UCD include user sketching, prototype evaluation, and usability experiments involving users.

> In the following sections, this report details techniques for creating a prototype, and how such a prototype can be used when performing a usability evaluation.

Reviewer: ```suggestion
The following sections detail techniques for creating a prototype, and how such a prototype can be used when performing a usability evaluation.
```

> In the following sections, this report details techniques for creating a prototype, and how such a prototype can be used when performing a usability evaluation.

Reviewer 2: Unclear what is meant here.

Are you saying that UCD provides the methods we will use going forward? Or?

When establishing a design for a product, one often begin with exploring initial thoughs and ideas for a design, only to then present the design to the user, make adjustments, and repeat the process. 
This can be described in process called the design funnel.\cite{BUXTON2007135_skething}

> Buxton\cite{BUXTON2007135_skething,BUXTON2007139_prototyping} describes how to move from exploratory analysis of different designs through skething to interactive, product refinement using prototypes.   

Reviewer: Should use citeauthor instead?

Reviewer 3: This has not been done other places and the formatting is wierd
